train.py

```diff
@@ -1311,16 +1311,16 @@ def update_training_scores_history(perf_scores_history, model, top1, top5, mAP,
 
             # Keep perf_scores_history sorted from best to worst
             if not args.sparsity_perf:
-                # Sort by mse as main sort key, then sort by epoch
+                # Sort by mse (1 - mse) as main sort key, then sort by epoch
```

Comment:
It seems, the code assumes to MSE be in range [0, 1] but it might not be. -MSE might be a better choice.